wallet-new/stack.yaml

```diff
@@ -0,0 +1,24 @@
+resolver: lts-9.1
```

Comment:
Woah, why is it a separate project instead of a subproject? I guess it's totally wrong. 
Nobody wants to deal with two `defaut.nix`. Also did you check that `pkgs/generate.sh` works correctly?